### PR TITLE
Extend Insights plan entity to support more actions

### PR DESCRIPTION
**Pull request by mirekdlugosz**
New actions supported:
* run Ansible playbook
* run Ansible playbook with REX customizations
* download Ansible playbook
* download plan CSV

Needed for https://github.com/SatelliteQE/robottelo/pull/8102 ; see it for test results.

### CODE REVIEW

**Review by pondrejk — APPROVED** [review]
ack pending travis

**Comment by mirekdlugosz** [comment]
Travis failures seem unrelated. They come from `airgun/widgets.py` and `airgun/views/activationkey.py`, neither of which I touched in this PR.

Both should be fixed in #528 , where Travis passes.

File: airgun/entities/rhai/plan.py [hunk 1]
```diff
@@ -40,6 +42,36 @@ def update(self, entity_name, values):
         view = PlanEditView(self.session.browser)
         view.fill_with(values, on_change=view.save.click)
 
+    def run_playbook(self, entity_name, customize=False, customize_values=None):
```
Comment (omkarkhatavkar) [inline airgun/entities/rhai/plan.py:45]: doc-strings are missing, just for consistency. non-blocking comment

**Review by omkarkhatavkar — APPROVED** [review]
ACK

File: airgun/entities/rhai/plan.py (hunk 1 above)
Comment (mirekdlugosz) [reply airgun/entities/rhai/plan.py:45]: Fair enough, let me add them real quick here

Comment (mirekdlugosz) [reply airgun/entities/rhai/plan.py:45]: Docstrings added in 53fd3c7e1e57539fb7a4cfb7ff1d45d44fe63de4 , which was amended and force-pushed

**Comment by omkarkhatavkar** [comment]
2 acks already done, waiting on travis to get complete

**Comment by omkarkhatavkar** [comment]
Travis Failure unrelated to this PR as mentioned earlier by Mirek, merging

**Pull request merged**